### PR TITLE
Change Aiven plugins/connectors description

### PR DESCRIPTION
**Pull request by willyborankin**
### Description

Change `Aiven` plugins/connectors description 

- Added `Encrypted Repository Plugin for OpenSearch` to community projects
- Added `Aiven Oy` icon to all plugins/connectors the `Aiven` team supports
- Added owner link to all plugins/connectors the `Aiven` team supports
 
### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by reta** [comment]
@krisfreedain mind taking a look please? Thank you!

File: _community_projects/encrypted-repository-opensearch-plugin.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+name: Encrypted Repository Plugin for OpenSearch
+description: Encrypted Repository for OpenSearch is a plugin that offers client-side encryption for snapshotting repositories and could be combined with all official supported repository plugins: File Systsem, Google Cloud Storage, Amazon S3 and Microsoft Azure.
```
Comment (krisfreedain) [inline _community_projects/encrypted-repository-opensearch-plugin.md:3]: when I build locally, it give this error:

"Error: YAML Exception reading /Users/freedak/Documents/OpenSearch/temp-willyborankin-add-encrypted-repository-opensearch-plugin-project-website/_community_projects/encrypted-repository-opensearch-plugin.md: (<unknown>): mapping values are not allowed in this context at line 3 column 195" 

also, shows as a blank tile locally on the "community_projects" page. Can you edit and push the PR once again?

**Review by krisfreedain — CHANGES_REQUESTED** [review]
an edit needed @willyborankin @reta

**Comment by krisfreedain** [comment]
thanks for the heads-up on this one @reta  - left some notes on the PR where an error came through. 

@nateynateynate @AMoo-Miki - can you keep an eye on this one? I'm OOO until June 3rd. Thanks.

File: _community_projects/encrypted-repository-opensearch-plugin.md (hunk 1 above)
Comment (willyborankin) [reply _community_projects/encrypted-repository-opensearch-plugin.md:3]: yes sorry will fix

**Comment by willyborankin** [comment]
I fixed descriptions for all connectors/plugins which Aiven team supports

**Review by nateynateynate — APPROVED** [review]
Looks good to me.

**Comment by willyborankin** [comment]
> Looks good to me.

BTW I added this PR https://github.com/opensearch-project/project-website/pull/849 to fix problems with ruby 3.x.

**Review by krisfreedain — APPROVED** [review]
looks great! Thank you @willyborankin

**Pull request merged**